pom.xml

```diff
@@ -27,6 +27,18 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
This adds Lombok to your project. For more information visit [lombok docs](https://projectlombok.org/setup/maven).